card.go

```diff
@@ -24,8 +24,8 @@ type CardFunding string
 // For more details see https://stripe.com/docs/api#create_card and https://stripe.com/docs/api#update_card.
 type CardParams struct {
 	Params
-	Token                                         string
-	Customer, Recipient                           string
+	External_Account, Token                       string
```

Comment:
Ah, I see. I think you you may have misreferenced the new field in `AppendDetails` below though. As far as I can tell, this is new used. Also, I don't think it's a string unfortunately ([create a card](https://stripe.com/docs/api/curl#create_card-source%20|%20external_account)).